### PR TITLE
Compass in overlay color

### PR DESCRIPTION
**Pull request by AlphaLima**

### CODE REVIEW

**Review by brunotl — COMMENTED** [review]
won't build :
`Common/Source/Draw/LKGeneralAviation.cpp:144:32: error: no matching function for call to ‘LKSurface::SelectObject(LKPen*&)’`

File: Common/Source/Utils.cpp [hunk 1]
```diff
@@ -175,6 +175,15 @@ void SetOverColorRef() {
...
             break;
     }
+
+ //   OverColorOutRef = OverColorRef.ContrastTextColor();
+
+    if(OverColorRef.Brightness() < 34)  // brightness belo 34%
+      OverColorOutRef = RGB_WHITE;
+    else
+      OverColorOutRef = RGB_BLACK;      
+
```
[the remark this answers is not in the document]
Comment (AlphaLima) [reply Common/Source/Utils.cpp:185]: tried it, did not work as good on darker colors. I nneded to make a function that give the same output colors like the other overlays. ContrastTextColor() did not do that.

[the remark this answers is not in the document]
Comment (AlphaLima) [reply Common/Source/Utils.cpp:185]: We can use it but we need taht for all other overlays as well.

[the remark this answers is not in the document]
Comment (brunotl) [reply Common/Source/Utils.cpp:185]: ContrastTextColor is not used ...

**Comment by brunotl** [comment]
Manually merged

**Pull request closed without merging**